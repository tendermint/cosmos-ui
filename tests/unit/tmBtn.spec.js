import { shallowMount } from "@vue/test-utils"
import htmlBeautify from "html-beautify"
import { TmBtn } from "@/index.js"

describe("TmBtn.vue", () => {
  let wrapper
  beforeEach(async () => {
    const value = "button"
    wrapper = shallowMount(TmBtn, {
      propsData: { value }
    })
  })

  it("has the expected html structure", () => {
    expect(htmlBeautify(wrapper.html())).toMatchSnapshot()
  })
})
