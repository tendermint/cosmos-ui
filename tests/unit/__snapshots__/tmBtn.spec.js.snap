// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TmBtn.vue has the expected html structure 1`] = `
<button class="tm-btn"><span class="tm-btn__container">
<!---->
<!---->
<span class="tm-btn__value">button</span></span>
</button>
`;
